File: LEETCODE/arrays/Easy/S8_LinerSearch.java
public class S8_LinerSearch {
    private static boolean linearsearch(int arr[], int key){
        int n = arr.length;

        for(int i = 0; i < n; i++){
            if(arr[i] == key){
                return  true;
            }
        }
        return false;
    }
    public static void main(String[] args) {
        int arr[] = {3,4,2,4,2,5,6};
        linearsearch(arr, 6);
    }
}

File: LEETCODE/arrays/Easy/S8_LinerSearch.cpp
#include<bits/stdc++.h>
using namespace std;

int linearsearch(int n, int k , vector<int> nums){
    for(int i = 0; i < n; i++){
        if(nums[i] == k){
            return i;
        }
    }
    return -1;
}
int main(){
    int n;
    int key;
    cin >> n >> key;
    vector<int>  nums(n);

    for(int i = 0; i < n; i++){
        cin >> nums[i];
    }
    linearsearch(n,key, nums);
    return 0;
}